be/src/vec/exec/join/vnested_loop_join_node.cpp

```diff
@@ -232,6 +231,9 @@ Status VNestedLoopJoinNode::sink(doris::RuntimeState* state, vectorized::Block*
 
 Status VNestedLoopJoinNode::push(doris::RuntimeState* state, vectorized::Block* block, bool eos) {
     COUNTER_UPDATE(_probe_rows_counter, block->rows());
+    _cur_probe_row_visited_flags.resize(block->rows());
```

Comment:
Use `_cur_probe_row_visited_flags.resize(block->rows(), 0);` to replace `std::fill`